docs/api-docs/partner/create-a-sandbox.md

```diff
@@ -0,0 +1,30 @@
+# Create a Sandbox Store
+
+### On this page
+- [Creating a sandbox store](#create-a-sandbox-store)
+- [Resources](#resources)
+</div> 
+
+## Creating a sandbox store
+
+1. Go to the [BigCommerce Pricing](https://www.bigcommerce.com/essentials/pricing/) page and select **Start your free trial**.
+
```

Comment:
@traci707 Can you remove the extra lines between each step? They cause extra spacing when the doc is rendered.